### ISSUE TITLE
filtering boolean values doesn't work

### ISSUE THREAD

**Issue by m11gt** [wenzhixin/bootstrap-table#2229]
I have a table with a Boolean column called vendor availability. I have a select filter with Yes = 1 and No = 0. When I filter for No, the results are received from the ajax call and displayed. When I filter for Yes, the results are received from the ajax call as you can see in the chrome developer tools, but the table shows no results.

![doesn t work](https://cloud.githubusercontent.com/assets/5625425/14874006/9cd1726e-0cc1-11e6-94e3-c46660e36cae.PNG)
![works](https://cloud.githubusercontent.com/assets/5625425/14874007/9ec6008a-0cc1-11e6-81bf-e36a50255063.PNG)

**Comment by djhvscf** [comment]
Can you provide an example, please? @m11gt

**Comment by m11gt** [comment]
I just did some more testing and realized that this only happens when I use the data-formatter attribute in the column I am filtering. If I removed the formatter that changes that 0 and 1 to X and Tick, the issue goes away.

The current site is setup on my local host, so I couldn't find an example with the same setup I have, but this jsfiddle example is the closest I can get to it.

http://jsfiddle.net/kb9ydyoz/2/

**Comment by m11gt** [comment]
Problem fixed :)

I just needed to add data-search-formatter='false' to the column with the issue.
Now the column looks like this

<th data-field='availability' data-sortable='true' data-formatter='status' data-filter-control='select' data-filter-data='var:boolean_filter' data-search-formatter='false'>Vendor Availability</th>

I found it in another issue resolution. The column filter extension needs better documentation.
